src/Worker.cs

```diff
@@ -34,18 +36,50 @@ public async static Task Main(string[] args)
                 .WithParsed(ops => arguments = ops)
                 .WithNotParsed(err => Environment.Exit(1));
 
+            // Create the very first Runspace so the debugger has the target to attach to.
+            // This PowerShell instance is shared by the first PowerShellManager instance created in the pool,
+            // and the dependency manager (used to download dependent modules if needed).
+            var firstPowerShellInstance = Utils.NewPwshInstance();
+            LogPowerShellVersion(firstPowerShellInstance);
+            WarmUpPowerShell(firstPowerShellInstance);
+
             var msgStream = new MessagingStream(arguments.Host, arguments.Port);
-            var requestProcessor = new RequestProcessor(msgStream);
+            var requestProcessor = new RequestProcessor(msgStream, firstPowerShellInstance);
 
             // Send StartStream message
-            var startedMessage = new StreamingMessage() {
+            var startedMessage = new StreamingMessage()
+            {
                 RequestId = arguments.RequestId,
                 StartStream = new StartStream() { WorkerId = arguments.WorkerId }
             };
 
             msgStream.Write(startedMessage);
             await requestProcessor.ProcessRequestLoop();
         }
+
+        // Warm up the PowerShell instance so that the subsequent function load and invocation requests are faster
+        private static void WarmUpPowerShell(System.Management.Automation.PowerShell firstPowerShellInstance)
+        {
+            // It turns out that creating/removing a function warms up the runspace enough.
+            // We just need this name to be unique, so that it does not coincide with an actual function.
+            const string DummyFunctionName = "DummyFunction-71b09c92-6bce-42d0-aba1-7b985b8c3563";
+
+            firstPowerShellInstance.AddCommand("New-Item")
```

Comment:
Same for `Microsoft.PowerShell.Management\\New-Item` and `Microsoft.PowerShell.Management\\Remove-Item`

Reply:
Done.